libs/atqr/domain/src/lib/domain-spec/stories/playerAdoptsGoal.disl

```diff
@@ -0,0 +1,27 @@
+read_also api.disl
+read_also enums.disl
+read_also ui.disl
+
+entity Goal
+  @persuer: User
```

Comment:
typo (pursuer)

Reply:
You should actually have a `@challenges` property and have `@pursers` be a dynamic property defined by `retun @challenges.map(c => playc.enrollments.player).flat()`